tests/test_downloader.py

```
@@ -4,13 +4,13 @@
 import shutil
 import time
 from pathlib import Path
-from unittest.mock import call, mock_open, patch
-
+from unittest.mock import mock_open, patch
 import pytest
 import requests
-
 from fetchtastic import downloader
 from fetchtastic.device_hardware import DeviceHardwareManager
+import os
```

Ruff (0.14.1)

Redefinition of unused `os` from line 2

Remove definition: `os`

(F811)